### PR TITLE
Change listitem according to guide

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #729
[Problem] Bad title on listview example, bad listview item font-size
[Solution] Change fontsize, change example header

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	<div class="ui-page">
 		<header class="ui-header">
 			<h2 class="ui-title">
-				Basic List
+				Title
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html:21]: @korneliakobiela  and @lslachciak 
I'd like to ask whch guideline we're following (html or pdf)? According to html we should have the following structure:
 - List
 - List: Main + Sub text
 - List: Main + Sub text(editable) 
 - List: Main + Sub text text_max

Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html:21]: In the HTML guide on `#artboard39`, you can see a list like this

File: src/css/profile/wearable/changeable/theme-circle/theme.layout.less [hunk 2]
```diff
@@ -4,7 +4,7 @@ Listview
 @list_item_max_height: 360 * @unit_base;
 @list_item_line_height: 73 * @unit_base;
 @list_item_anchor_padding: 9 * @unit_base 30 * @unit_base 8 * @unit_base;
-@font_size_list_item: 36 * @unit_base;
+@font_size_list_item: 38 * @px_base;
```
Comment (grzegorz-czajkowski) [inline src/css/profile/wearable/changeable/theme-circle/theme.layout.less]: size is ok but I think we should use unit_base according to this file.

Comment (korneliakobiela) [reply src/css/profile/wearable/changeable/theme-circle/theme.layout.less]: @grzegorz-czajkowski why? I talked about it on chat, and there was information that now we use `@px_base` and `@sp_base`. Is it true only for the mobile profile?

Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/theme.layout.less]: Ok.

Comment (lslachciak) [reply src/css/profile/wearable/changeable/theme-circle/theme.layout.less]: I think we should define and use sp_base in case of fonts

Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/theme.layout.less]: Please define and use sp_base according to conversation.

Comment (korneliakobiela) [reply src/css/profile/wearable/changeable/theme-circle/theme.layout.less]: Done!

**Review by grzegorz-czajkowski — APPROVED** [review]

**Review by lslachciak — APPROVED** [review]

**Pull request merged**